src/flight/filter.c

```diff
@@ -90,6 +90,103 @@ float filter_lp_pt3_step(filter_lp_pt3 *filter, filter_state_t *state, float in)
   return state->delay_element[0];
 }
 
+void filter_lp_lulu_init(filter_t *filter, filter_lulu_state_t *lulu_state, float hz) {
+  //The window value is half the wavelength of the wave that it filters.  So if the wavelength of the cutoff frequency is 2 samples, the N value should be 1.  If the wavelength is 4, N should be 2.  Etc.
+  float cutoff_wave_length = 1.0f / hz / 4.0f;
+  float loop_wave_length = state.looptime_autodetect * 1e-6f;
+  int window_half_length = cutoff_wave_length / loop_wave_length;
+  
+  filter->lp_lulu.num_samples = constrain(window_half_length, 1, 12);
+  filter->lp_lulu.window_size = filter->lp_lulu.num_samples * 2 + 1;
+  (*lulu_state).window_buf_index = 0;
+
+  memset((*lulu_state).interim, 0, sizeof(float) * (filter->lp_lulu.window_size));
+  memset((*lulu_state).interim_b, 0, sizeof(float) * (filter->lp_lulu.window_size));
+}
+
+float fix_road(float *series, float *series_b, int index, int filter_n, int window_size) {
```

Comment:
```suggestion
static float fix_road(float *series, float *series_b, int index, int filter_n, int window_size) {
```

Reply:
I'll make the change, test, and then commit the change if it works! I expect it to work, but I want to make absolutely sure :)